coccinelle/run-coccinelle.sh

```diff
@@ -10,6 +10,7 @@ EXCLUDED_PATHS=(
 )
 
 top="$(git rev-parse --show-toplevel)"
+headers="$(mktemp)"
```

Comment:
This variable seems unused. Left over from previous version?

Reply:
Quite indeed, thanks for catching this.